jupyter_client/connect.py

```
@@ -152,6 +152,7 @@ def write_connection_file(
         control_port=control_port,
         hb_port=hb_port,
     )
+    a = 10//0
```

Ruff

Local variable `a` is assigned to but never used

Remove assignment to unused variable `a`

(F841)